### DISCUSSION TITLE
Plan to allow multiple Attendees

### DISCUSSION THREAD

**Discussion by Harmonickey** [add2cal/add-to-calendar-button#595] [Ideas / Requests]
I see that the library requires that attendee is only one NAME|EMAIL pair.  Which I see is certainly true for the RFC defined ATTENDEE field.  But the RFC also defines that you can add multiple distinct ATTENDEE fields so that there can be more than just one attendee added to the calendar event.

https://www.ietf.org/rfc/rfc2445.txt

ORGANIZER:MAILTO:jsmith@host1.com
ATTENDEE;MEMBER="MAILTO:DEV-GROUP@host2.com":
   MAILTO:joecool@host2.com
ATTENDEE;DELEGATED-FROM="MAILTO:immud@host3.com":
   MAILTO:ildoit@host1.com

Do you have this on the horizon to add?

**Comment by jekuer** [comment]
At the moment, having the attendee set can be necessary to make an event show up as "accepted" in the calendar.
So far, we did not see a use case for multiple attendees. What's yours?

**Reply by jekuer** [reply]
Also see #380
